plane-detection.bs

```diff
@@ -183,6 +184,14 @@ Each {{XRPlane}} has an associated <dfn for=XRPlane>frame</dfn>.
 
 The {{XRPlane/polygon}} is an array of vertices that describe the shape of the plane. They are returned in the form of a loop of points at the edges of the polygon, expressed in the coordinate system defined by {{XRPlane/planeSpace}}. The Y coordinate of each vertex MUST be <code>0.0</code>.
 
+<div class="unstable">
+
+The {{XRPlane/semanticLabels}} is an array of strings that describe the semantic label of the polygon. This array is empty if there is no semantic information.
+
+ISSUE: should we predefine these labels or have a separate profile?
```

Comment:
Something akin to input profiles would probably make sense here since IMO it'd increase the chances of multiple implementations agreeing on the same labels, otherwise I'm not sure how site authors could sanely write a cross-platform experience?

Reply:
I agree. It shouldn't be part of the spec so we can add them anytime we want.

Reply:
Also, I made them a list to future proof but at least for Meta, there is only 1 label.